docs/glossary/u/Unicode.lcdoc

```diff
@@ -5,11 +5,11 @@ Synonyms: unicode
 Type: glossary
 
 Description:
-A standard for representing <character|characters> as <2-byte> numbers,
+A standard for representing <double-byte character|double-byte characters>,
```

Comment:
This is not quite the same as the original definition. I suggest:
- keep the original definition
- replace `<2-byte>` with `2-byte`
- remove ` 2-byte (glossary)` from the references, as there is no such entry